airbyte-integrations/connectors/source-sftp-bulk/source_sftp_bulk/spec.json

```diff
@@ -81,6 +81,30 @@
         "examples": ["/logs/2022"],
         "order": 8
       },
+      "max_depth": {
+        "title": "Recursion depth (Optional)",
+        "description": "Number of levels to search for files in the folder path",
```

Comment:
Add a comment saying the code use inclusive `max_depth` so people know if they insert 10 The code will go to 10 and end.

Reply:
if max_depth=10 the code will look into nested folders up to the 10th level down in the tree. An option would be to cap the value to the range [0,2]